rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -31,8 +34,16 @@ elasticsearch_apt_repos:
 elasticsearch_apt_keys:
   - { url: "http://packages.elastic.co/GPG-KEY-elasticsearch", state: "present" }
 
+elasticsearch_version: 2.4.1
```

Comment:
It's more for the version check than anything, I'm fine with not installing specific versions.